packages/teleport/src/Teleport.tsx

```diff
@@ -17,10 +17,14 @@ limitations under the License.
 import React from 'react';
 import ThemeProvider from 'design/ThemeProvider';
 
+import { getEnterpriseFeatures } from 'e-teleport/features';
```

Comment:
this would pull `e-teleport` as a dependency when we build OSS?

Reply:
Aha I just noticed this whilst rebuilding to test the OSS version. Cheers.